config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,22 @@
+CarrierWave.configure do |config|
+  if Rails.env.test?
+    config.storage = :file
+    config.enable_processing = false
+  else
+    config.storage = :fog
+    config.fog_credentials = {
+      provider:              'AWS',
+      aws_access_key_id:     ENV['AWS_ACCESS_KEY_ID'],
+      aws_secret_access_key: ENV['AWS_SECRET_ACCESS_KEY'],
+      region:                ENV['AWS_REGION']
+    }
+
+    if Rails.env.production?
+      config.fog_directory  = ENV['AWS_S3_BUCKET_PRODUCTION']
+    elsif Rails.env.staging?
+      config.fog_directory  = ENV['AWS_S3_BUCKET_STAGING']
+    elsif Rails.env.development?
```

Comment:
This is certainly not a recommended way. If i would work with you - I would need to build my own AWS bucket because you hide the key and bucket.